### PR TITLE
[docs] C# Entity framework quickstart page

### LINKED ISSUE #10635: [doc] Document EntityFramework in Quickstart
Here is the link to the [initial documentation](https://docs.google.com/document/d/1Wbd9S6G5uXEH9w4NCyZClMljPYCq_sZCBIVEsEYgheo/edit?usp=sharing) of EntityFramework to add in the [quickstart - build application tab](https://docs.yugabyte.com/latest/quick-start/build-apps/csharp/ysql/). In case of any questions please let me know.

### PR DESCRIPTION
**Pull request by aishwarya24**
Resolves #10635 

@netlify /latest/quick-start/build-apps/csharp/ysql-entity-framework/

### CODE REVIEW

File: docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+        "email": "jsmith@example.com"
+      },
+      "userId": null,
+      "orderTotal": 15,
+      "products": []
+    }
+  ],
+  ...
+}
+```
```
Comment (Sfurti-yb) [inline docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md:309]: This is the old output. I have updated the new one in the doc(there are some differences with the format and the fields that are output). Other than this, LGTM.

Comment (aishwarya24) [reply docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md:309]: Thanks @Sfurti-yb , I'll update it.

File: docs/content/latest/quick-start/build-apps/csharp/ycql.md [hunk 2]
```diff
@@ -20,7 +20,12 @@ showAsideToc: true
       YSQL
     </a>
   </li>
-
+ <li>
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/csharp/ycql.md:23]:
```suggestion
  <li>
```

File: docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+(1 row)
+```
+
+### Using the REST API
+
+```sh
+$ curl http://localhost:8080/users
+```
+
+```json
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md:229]:
```suggestion
```output.json
```

File: docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+  ],
+  ...
+}
+```
+
+```sh
+$ curl http://localhost:8080/products
+```
+
+```json
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md:253]:
```suggestion
```output.json
```

File: docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+  ],
+  ...
+}
+```
+
+```sh
+$ curl http://localhost:8080/orders
+```
+
+```json
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md:277]:
```suggestion
```output.json
```

**Review by polarweasel — APPROVED** [review]
LGTM... make the little changes and let's get it published

**Comment by aishwarya24** [comment]
Thanks @Sfurti-yb and @polarweasel . I resolved the comments and also the indentation.

**Pull request merged**